pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -8511,3 +8511,78 @@ spec:
 func lessTaskResourceBindings(i, j v1beta1.TaskResourceBinding) bool {
 	return i.Name < j.Name
 }
+
+func TestReconcile_ApplyTaskLevelComputeResources(t *testing.T) {
+	multiStepsTask := parse.MustParseTask(t, `
+metadata:
+  name: foo-task
+  namespace: default
+spec:
+  steps:
+  - name: 1st-step
+    image: foo-image
+    command: 
+    - /foo-cmd
+  - name: 2nd-step
```

Comment:
nit: could you scope the test case down to what it's actually trying to test? this doesn't need multiple steps, and the name of this variable could be more descriptive of what the test is trying to test.

Reply:
agree. let me update.